backend/coreapp/views/scratch.py

```diff
@@ -461,11 +461,17 @@ def fork(self, request: Request, pk: str) -> Response:
 
         ser = ScratchSerializer(data=fork_data, context={"request": request})
         ser.is_valid(raise_exception=True)
+
+        libraries = [
+            Library(name=lib["name"], version=lib["version"])
```

Comment:
Do we expect different fields or could you pass `**lib`?

Reply:
I'd normally say that I prefer to be explicit, but since we're going from Model -> serializer -> Model, I guess we have bigger problems if those are out of sync ..so sounds good :)